pkg/projects/descriptions.json

```diff
@@ -20,8 +25,23 @@
         "CommonTypes": [ ]
     },
     {
+        "Name": "Microsoft.NETCore",
+        "Description": "Provides a set of packages that can be used when building portable libraries on .NETCore based platforms.",
+        "CommonTypes": [ ]
+    },
+    {
         "Name": "Microsoft.NETCore.App",
         "Description": "A set of .NET API's that are included in the default .NET Core application model.",
         "CommonTypes": [ ]
     },
+    {
+        "Name": "Microsoft.NETCore.Console",
```

Comment:
Currently there isn't a direct release ship between these. .NETCore.Console was a precursor to .NETCore.App. Microsoft.NETCore package is used by PCL projects but might be superseded by NETStandard.Library. For now my main intent is to get these building somewhere in the open so we don't start publishing bogus versions of these when we switch to shipping from the open. I will not be surprised at all if we end up deleting some of these later.